nilearn/glm/first_level/first_level.py

```diff
@@ -873,18 +883,15 @@ def first_level_from_bids(dataset_path, task_label, space_label=None,
     if not isinstance(img_filters, list):
         raise TypeError('img_filters must be a list, instead %s was given' %
                         type(img_filters))
-    for img_filter in img_filters:
-        if (not isinstance(img_filter[0], str)
-                or not isinstance(img_filter[1], str)):
+    for this_filter in img_filters:
```

Comment:
changed the variable name, to make it easier to distinguish between `img_filters` and `img_filter`.